src/js/messaging/reducers/folders.js

```diff
@@ -27,17 +26,11 @@ const initialState = {
       attributes: {},
       filter: {},
       messages: [],
-      pagination: {
-        currentPage: 0,
-        perPage: 0,
-        totalEntries: 0,
-        totalPages: 0
-      },
-      persistFolder: 0,
+      pagination: {},
```

Comment:
I'm going to revert to the initial 0 values for now and follow up with a PR to handle undefined/null states for the relevant components.